com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -235,6 +235,27 @@ public void UnregisterSideChannel(SideChannel channel)
             Communicator?.UnregisterSideChannel(channel);
         }
 
+        /// <summary>
+        /// Returns the SideChannel of Type T if there is one registered, or null if it doesn't.
+        /// If there are multiple SideChannels of the same type registered, only the first found will be returned.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
+        public T GetSideChannel<T>() where T: SideChannel
+        {
+            return Communicator.GetSideChannel<T>();
+        }
+
+        /// <summary>
+        /// Returns all SideChannels of Type T that are registered.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
+        public List<T> GetSideChannels<T>() where T: SideChannel
```

Comment:
Method name is too close to GetSideChannel IMO. I would say let's keep only this one as it is more general.

Reply:
Discussed offline; should be fine since this is what MonoBehavior does for GetComponent.